### ISSUE TITLE
Question. When I debug in Chrome inspect, I get an error in `indexedDB`.

### ISSUE THREAD

**Issue by nabezokodaikon** [denoland/deno#7002]
How can I use `indexedDB` for Chrome inspect debugging?

Sample code
```typescript
export class ProductDatabase {
  constructor(private name: string, private version: number) {
    const openDatabaseRequest = indexedDB.open(this.name, this.version);
    openDatabaseRequest.onupgradeneeded = this.upgrade;
  }
}
```

Error
```bash
error: Uncaught ReferenceError: indexedDB is not defined
    const openDatabaseRequest = indexedDB.open(this.name, this.version);
```

Running command
```bash
$ deno run --inspect-brk --allow-read --allow-net --config tsconfig.json db.ts
```

tsconfig.json
```json
{
  "compilerOptions": {
    "lib": [
      "deno.ns",
      "dom"
     ],
    "plugins": [
      {
        "name": "typescript-deno-plugin"
      }
    ]
  }
}
```

**Comment by crowlKats** [comment]
IndexedDB is not yet implemented in Deno, see #1699

**Comment by nabezokodaikon** [comment]
Thank you for your reply. understood.
